main.go

```diff
@@ -100,8 +100,8 @@ func main() {
 		},
 	}
 
-	root.PersistentFlags().StringVarP(&logLevel, "loglevel", "l", "warn", "amount of information outputted (debug, info, warn, error)")
-	root.PersistentFlags().StringVarP(&outputMode, "output", "o", oktetoLog.TTYFormat, "output format (tty, plain, json)")
+	root.PersistentFlags().StringVarP(&logLevel, "log-level", "l", "warn", "amount of information outputted (debug, info, warn, error)")
+	root.PersistentFlags().StringVar(&outputMode, "log-output", oktetoLog.TTYFormat, "output format for logs (tty, plain, json)")
```

Comment:
We can't change this without coordinating with the team. The pipeline installer depends on this flag to run the pipelines. I would avoid changing this flag as much as possible but we can change it if we coordinate with the team. WDYT @ifbyol @pchico?

Reply:
If we keep this flag the same, the changes in the PR should still work - only that `okteto context help` would not mention the new `output=json/yaml` flag.

Reply:
@jLopezbarb @ifbyol let's do this change, `-o` is a very common flag for output format.
Since this is more like an internal flag, we can easily change the installer when upgrading the cli in the installer.
For example, buildkit uses `--progress` for this.

Reply:
Ok, as soon as these changes are in a CLI release, we can update the pipeline installer to pass the new flag. It is also something that we should see easily when upgrading the CLI in the backend because logs wouldn't display properly

Reply:
@RinkiyaKeDad to reduce overhead on monday's release, let's merge this right after releasing CLI 2.0.2

Reply:
@RinkiyaKeDad green light to merge whenever you are ready :-)